src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
> could we clear the error list and add all elemens from the errorMap into the list? I mean the errorMap contains everything but the errorList doesn't, right?

That seems the right way. Yes the errorMap contains all needed errors, we only have to map it to an array and assign the result to `this.errorList`

> I am wondering that we can see all validation error messages on the fields but only the invalid indicators/borders are missing...

Yes, only the highlight was removed. This is caused due to `valideElements` return all elements except the one the remote rule was attached to. ( see [core.js#L796-L800](https://github.com/jzaefferer/jquery-validation/blob/master/src/core.js#L796-L800) and [core.js#L806-L814](https://github.com/jzaefferer/jquery-validation/blob/master/src/core.js#L806-L814) )